pokedex/src/Components/Aboutme/Aboutme.jsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import "../Aboutme/Aboutme.css";
+import pokeball from "../../assets/pokeball.png";
+import { Link } from "react-router-dom";
+
+const Aboutme = () => {
+  return (
+    <div className="about-me">
+      <Link to="/">
+        <img alt="pokeball" className="header__ball" src={pokeball} />
```

Comment:
According to the checklist and criteria, you need to use `BEM` in your project. Please, correct the css selectors according to BEM